sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -225,18 +225,18 @@ private static void CloseLink(RequestResponseAmqpLink link)
         }
 
         /// <summary>
-        /// Receives a batch of <see cref="ServiceBusMessage" /> from the Service Bus entity.
+        /// Receives a list of <see cref="ServiceBusMessage" /> from the Service Bus entity.
         /// </summary>
         ///
-        /// <param name="maxMessages">The maximum number of messages to receive in this batch.</param>
+        /// <param name="maxMessages">The maximum number of messages to receive.</param>
         /// <param name="maxWaitTime">An optional <see cref="TimeSpan"/> specifying the maximum time to wait for the first message before returning an empty list if no messages have been received.
         /// If not specified, the <see cref="ServiceBusRetryOptions.TryTimeout"/> will be used.</param>
         /// <param name="timeout">The per-try timeout specified in the RetryOptions.</param>
         /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
         ///
-        /// <returns>The batch of <see cref="ServiceBusMessage" /> from the Service Bus entity this receiver is associated with. If no messages are present, an empty list is returned.</returns>
+        /// <returns>The list of <see cref="ServiceBusMessage" /> from the Service Bus entity this receiver is associated with. If no messages are present, an empty list is returned.</returns>
```

Comment:
I went back and forth on this but since it returns an IList, list feels appropriate. "Set" might also mean a concrete implementation to some. Alternatively, I can just cref the whole IList but that felt unnecesary.

Reply:
I'm good whichever way you feel is best.

Reply:
This might be something to look at in terms of consistency across the Track 2 libraries, as this surely isn't a new issue.